### PR TITLE
Use LinkedIn v2 API

### PR DESCRIPTION
**Pull request by pandu-crunchbase**
This looks larger than it is -- basically three lines of source code (in `src/modules/linkedin.js`) have changed.

### CODE REVIEW

File: src/modules/linkedin.js [hunk 1]
```diff
@@ -7,15 +7,15 @@
...
+				auth: 'https://www.linkedin.com/oauth/v2/authorization',
+				grant: 'https://www.linkedin.com/oauth/v2/accessToken'
 			},
 
 			// Refresh the access_token once expired
 			refresh: true,
 
 			scope: {
-				basic: 'r_basicprofile',
+				lite: 'r_liteprofile',
```
Comment (justinappler) [inline src/modules/linkedin.js]: I thought that `basic` was sort of a reserved keyword in Hello? Doesn't it get included by default if it exists? It seems like the behavior most people would want is for this to be `basic: 'r_liteprofile'` so basic continues to 'just work'.

**Review by justinappler — CHANGES_REQUESTED** [review]

File: src/modules/linkedin.js (hunk 1 above)
Comment (pandu-crunchbase) [reply src/modules/linkedin.js]: Oh yeah, I think you're right. Good catch! Thanks.

**Review by justinappler — APPROVED** [review]

**Comment by pandu-crunchbase** [comment]
Thanks Justin! I'm going to do some QAing before merging.

**Comment by pandu-crunchbase** [comment]
OK, looks good! Merging.

I'm going to use a merge commit (instead of squash or rebase) here -- I don't think it'll make a big difference either way, but I don't expect we'll be doing a lot of changes in this repo so the added clarity (at the cost of brevity) in the commit history of a merge commit seems good.

**Pull request merged**